go-controller/pkg/metrics/ovnkube_controller.go

```diff
@@ -273,17 +274,37 @@ var metricEgressFirewallCount = prometheus.NewGauge(prometheus.GaugeOpts{
 var metricANPCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Namespace: MetricOvnkubeNamespace,
 	Subsystem: MetricOvnkubeSubsystemController,
-	Name:      "admin_network_policy_custom_resource_total",
+	Name:      "admin_network_policy_total",
 	Help:      "The total number of admin network policies in the cluster",
 })
 
 var metricBANPCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Namespace: MetricOvnkubeNamespace,
 	Subsystem: MetricOvnkubeSubsystemController,
-	Name:      "baseline_admin_network_policy_custom_resource_total",
+	Name:      "baseline_admin_network_policy_total",
 	Help:      "The total number of baseline admin network policies (0 or 1) in the cluster",
```

Comment:
not sure what `(0 or 1)` means here

Reply:
its a singleton object, so there can be only 1 max OR none. Perhaps I can use the Set(1) and/or Set(0) instead here.

Reply:
the wording is confusing for me (at least). IMHO it could be a code comment but it doesn't help in the metric help.

Reply:
ack sg let me remove this from the help you are right, adds more confusion than anything else.